build/cqs.tsv

```diff
@@ -101,20 +104,22 @@ it.geosolutions.jaiext.orderdither:jt-orderdither	1.0.16	compile
 it.geosolutions.jaiext.piecewise:jt-piecewise	1.0.16	compile
 it.geosolutions.jaiext.rescale:jt-rescale	1.0.16	compile
 it.geosolutions.jaiext.rlookup:jt-rlookup	1.0.16	compile
+it.geosolutions.jaiext.scale2:jt-scale2	1.1.1	compile
 it.geosolutions.jaiext.scale:jt-scale	1.0.16	compile
+it.geosolutions.jaiext.shadedrelief:jt-shadedrelief	1.1.1	compile
```

Comment:
these versions not matching the other jaiext jars makes me think we need to bump the version in our pom

Reply:
fixed